script/bulk_import/base.rb

```diff
@@ -1225,6 +1243,7 @@ def process_group(group)
 
     group[:created_at] ||= NOW
     group[:updated_at] ||= group[:created_at]
+    group[:assignable_level] ||= Group::ALIAS_LEVELS[:nobody] if @assign_plugin_enabled
```

Comment:
I think it's safe to set this unconditionally. There's no need for `@assign_plugin_enabled` in that case. A comment might be enough.